ctms/acoustic_service.py

```diff
@@ -253,6 +253,13 @@ def _main_table_converter(self, contact, main_fields):
                         self.context["trace"] = inner_value
 
                     if acoustic_field_name in main_fields:
+                        # TODO: The create_timestamp is currently a Date field in Acoustic.
+                        # Sending time information will prevent Acoustic to set the value properly.
+                        # We should configure Acoustic to use a Timestamp for this data.
+                        # This is being tracked in https://github.com/mozilla-it/ctms-api/issues/803
```

Comment:
This is the issue for migrating "boolean-like" columns to actual booleans.

Should we make a new issue or expand the scope of #803 to track migrating all "date-like" columns to dates or timestamps?